src/crypto/tls/handshake_client_tls13.go

```diff
@@ -815,7 +826,7 @@ func (hs *clientHandshakeStateTLS13) abortIfRequired() error {
 	if c.ech.offered && !c.ech.accepted {
 		// If ECH was rejected, then abort the handshake.
 		c.sendAlert(alertECHRequired)
-		return errors.New("tls: ech: rejected")
+		return errors.New("tls: ech: rejected" + string(base64.StdEncoding.EncodeToString(c.ech.retryConfigs)))
```

Comment:
`string(...)` is redundant as `EncodeToString` already returns one. Alternatively:
```suggestion
		return fmt.Errorf("tls: ech: rejected %s", base64.StdEncoding.EncodeToString(c.ech.retryConfigs))`
```
Is it really important to log this or can this addition be dropped?

Reply:
Deja Vu :)

This was done for testing and wasn't meant to be merged. Reverted.